src/include/dhcp-server/widgets.rb

```diff
@@ -1054,15 +1009,19 @@ def InitWidgets
           #FIXME CWM should be able to handle virtual widgets
           "widget"        => :textentry
         },
-        "start"                => {
-          "widget" => :checkbox,
-          # check box
-          "label"  => _("&Start DHCP Server"),
-          "help"   => Ops.get(@HELPS, "start", ""),
-          "init"   => fun_ref(method(:startInit), "void (string)"),
-          "handle" => fun_ref(method(:startHandle), "symbol (string, map)"),
-          "store"  => fun_ref(method(:startStore), "void (string, map)"),
-          "opt"    => [:notify]
+        "service_status"         => {
+          "widget" => :custom,
+          "custom_widget" => @status_widget.widget,
+          "help"   => @status_widget.help,
+          "init"   => fun_ref(method(:init_service_status), "void (string)"),
+          "handle" => fun_ref(method(:handle_service_status), "symbol (string, map)"),
+          "store"  => fun_ref(method(:store_service_status), "void (string, map)")
+        },
+        "apply"           => {
+          "widget" => :custom,
```

Comment:
push button is supported as widget by CWM - https://github.com/yast/yast-yast2/blob/630cf98148facf439a4439c189d610ba031d1537/library/cwm/src/modules/CWM.rb#L609